packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-form.tsx

```diff
@@ -9,7 +9,7 @@ import {
     MobileWrapper,
     SelectNative,
 } from '@deriv/components';
-import { useFileUploader } from '@deriv/hooks';
+// import { useFileUploader } from '@deriv/hooks';
```

Comment:
can we remove the comment if not needed ?

Reply:
Yes thanks!